docs/guide/markdown.md

```diff
@@ -79,6 +79,8 @@ meta:
 
 :tada: :100:
 
+[A list of all emojis available can be found here.](https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/full.json)
```

Comment:
Only linked "here" would looks better:

```md
A list of all emojis available can be found [here](https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/full.json).
```

A list of all emojis available can be found [here](https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/full.json).